src/main/resources/com/cloudbees/hudson/plugins/folder/AbstractFolder/configure.jelly

```diff
@@ -78,7 +75,7 @@ THE SOFTWARE.
             The save/apply buttons are out of place until the DOM is modified by expanding config elements or by
             resizing the browser window.
             -->
-            <f:textbox clazz="block-control" checkUrl="checkDisplayName" checkDependsOn="jobName"/>
+            <f:textbox clazz="block-control" checkUrl="checkDisplayName" checkDependsOn=""/>
```

Comment:
```suggestion
            <f:textbox clazz="block-control" checkUrl="checkDisplayName"/>
```

or as previously mentioned simply
```suggestion
            <f:textbox clazz="block-control"/>
```

if you use the conventional check method name.

Also, this would be a good time to revisit the TODO comment above and maybe delete `clazz` as well if that is obsolete.

Reply:
hm, I tried to apply both suggestions but for some reason they don't works

1. When I try `<f:textbox clazz="block-control" checkUrl="checkDisplayName"/>` the page becomes a blank screen. 
2. And when I try `<f:textbox clazz="block-control"/>` view displays well, but checkDisplayName request does not send

Reply:
Not sure offhand, but form validation works fine in core and tons of plugins without overriding conventional values in attributes.

Reply:
>Also, this would be a good time to revisit the TODO comment above and maybe delete clazz as well if that is obsolete.

Remove block-control class and tested on Chrome and Firefox

Reply:
What I found, based on this [doc](https://www.jenkins.io/doc/developer/security/csp/#legacy-javascript-checkurl-validation)  `checkDependsOn` must exists but could be empty, as of Jenkins 2.360

Reply:
I do not know why the doc says that, since there are tons of examples of form fields which implicitly use a `doCheckXXX` method without overriding either `checkUrl` or `checkDependsOn`, including some using `@AncestorInPath`.See https://github.com/jenkinsci/jenkins/blob/ccc0878ba856d18bb0fb9cffab61bfc69096efd1/core/src/main/java/jenkins/triggers/ReverseBuildTrigger.java#L209 + https://github.com/jenkinsci/jenkins/blob/ccc0878ba856d18bb0fb9cffab61bfc69096efd1/core/src/main/resources/jenkins/triggers/ReverseBuildTrigger/config.jelly#L28-L29 for one example; `TimerTrigger.spec` is another. I _suspect_ that your problem is that you are still using a nonstandard `checkUrl`. Try omitting it, and using the conventional check method name.

Reply:
CC @daniel-beck as author of https://github.com/jenkins-infra/jenkins.io/pull/5301, which links to https://github.com/jenkinsci/jenkins/pull/6857 that looks much too complicated to me: https://github.com/jenkinsci/jenkins/blob/ccc0878ba856d18bb0fb9cffab61bfc69096efd1/core/src/main/java/hudson/slaves/SimpleScheduledRetentionStrategy.java#L261 is not using a conventional name `doCheckStartTimeSpec`; given that, I think https://github.com/jenkinsci/jenkins/blob/ccc0878ba856d18bb0fb9cffab61bfc69096efd1/core/src/main/resources/hudson/slaves/SimpleScheduledRetentionStrategy/config.jelly#L28-L31 could be greatly simplified to regular form binding (other than a custom `help` to avoid duplication)

```xml
<f:entry field="startTimeSpec" title="${%Startup Schedule}"  help="/descriptor/hudson.triggers.TimerTrigger/help/spec">
    <f:textarea/>
</f:entry>
```

Reply:
Made it work by moving doCheck to descriptor. See [efce1f8](https://github.com/jenkinsci/cloudbees-folder-plugin/pull/268/commits/efce1f896631e15a5126151af3a606f26c742f14)